src/Numerics/Mesh/Elements.jl

```diff
@@ -115,4 +115,41 @@ function interpolationmatrix(
     I
 end
 
+"""
+    function jacobip(α::Int, β::Int, np::Int, x::AbstractVector)
+
+    Returns the `np+1` Jacobi polynomials, with parameter `(α, β)`, of order `np` on 1D grid `x`.
```

Comment:
```suggestion
Returns the `np+1` Jacobi polynomials, with parameter `(α, β)`, of order `np` on 1D grid `x`.
```

- This shouldn't be indented (indented code is rendered as a code block)
- This isn't a useful description: it returns an array, not a polynomial. What is in the array that it returns?